app/controllers/assignments_controller.rb

```diff
@@ -616,45 +616,28 @@ def view_summary
   end
 
   def download_assignment_list
-    assignments = Assignment.all
-
+    @output = Assignment.get_assignment_list(params[:file_format], DEFAULT_FIELDS)
```

Comment:
Make `output` a *local variable* (don't put the `@` in front of it). In general, the `@` should be used rarely, only when the variable needs to be passed to the view.

Also a reminder to move `DEFAULT_FIELDS` into the model.